pkg/instrumentation/metrics.go

```diff
@@ -45,6 +51,12 @@ type Metrics struct {
 	OrgTotal                                       prometheus.Gauge
 	RHCertExpiryDays                               prometheus.Gauge
 	RHReposSnapshotNotCompletedInLast36HoursCount  prometheus.Gauge
+	TemplatesUpdateTaskPendingTimeAverage          prometheus.Gauge
```

Comment:
I could see this being a GaugeVector of pending times, partitioned by the task type. That may simplify doing this again for another task type. Not sure if it would work, but might be worth trying?

Reply:
Good idea, I like it 😁 Implemented it so have a look again at how it is now. 😅
Right now it does it for all task types, which can be changed or left as is, since it's still only one query.

Reply:
good idea!

Reply:
just gonna test now